src/rust/graph-generator-lib/src/lib.rs

```diff
@@ -30,26 +30,6 @@ use sqs_lambda::event_handler::EventHandler;
 use sqs_lambda::local_sqs_service::local_sqs_service;
 use std::str::FromStr;
 
-#[derive(Clone, Default)]
-pub struct ZstdJsonDecoder {
-    pub buffer: Vec<u8>,
-}
-
-impl<D> PayloadDecoder<D> for ZstdJsonDecoder
-where
-    for<'a> D: Deserialize<'a>,
-{
-    fn decode(&mut self, body: Vec<u8>) -> Result<D, Box<dyn std::error::Error>> {
-        self.buffer.clear();
-
-        let mut body = Cursor::new(&body);
-
-        zstd::stream::copy_decode(&mut body, &mut self.buffer)?;
-
-        Ok(serde_json::from_slice(&self.buffer)?)
-    }
-}
-
```

Comment:
This was unused and seems to have a sister implementation in [generic-subgraph-generator](https://github.com/grapl-security/grapl/blob/staging/src/rust/generic-subgraph-generator/src/main.rs#L628) where it is actually used.